packages/expo-image/src/ExpoImage.tsx

```diff
@@ -17,7 +17,37 @@ export default function ExpoImage({ source, style, ...props }: ImageProps) {
   // additional layout passes, which we want to prevent.
   if (!Array.isArray(resolvedSource)) {
     const { width, height } = resolvedSource;
-    resolvedStyle = StyleSheet.flatten([{ width, height }, style]);
+    resolvedStyle.width = resolvedStyle.width ?? width;
+    resolvedStyle.height = resolvedStyle.height ?? height;
+  }
+
+  // Shadows behave different on iOS, Android & Web.
+  // Android uses the `elevation` prop, whereas iOS
+  // and web use the regular `shadow...` props.
+  let hasShadows = false;
+  if (Platform.OS === 'android') {
+    delete resolvedStyle.shadowColor;
```

Comment:
Do you think we should ask the users to use `Platform.select` to be explicit about the props for different platforms? (So it's immediately visible that shadows on Android are not handled by `shadow*` styles?) (API design question, will cover in the API doc)

Reply:
Do you think it would be possible to support `shadowColor` some way (one way I found was described in [this article](https://medium.com/@ArmanSo/take-control-of-views-shadow-android-c6b35ba573e9))?

Should we research it right now (and it's performance), or would you like to try it later (or never try it)?

Reply:
Yes that's actually a really good one. It would be really nice to support that.